### ISSUE TITLE
Add extraction of validation set

### ISSUE THREAD

**Issue by ERUD1T3** [ERUD1T3/decision-tree#9]
- [x] add a way to extract the validation set from the training set
